app/validators/preprint-form-validator.js

```diff
@@ -0,0 +1,38 @@
+import { validator, buildValidations } from 'ember-cp-validations';
+
+const Validations = buildValidations({
```

Comment:
Right, but couldn't you define it at the top of the controller instead? (instead of importing it)